server/closed_conns_test.go

```diff
@@ -267,7 +267,7 @@ func TestClosedMaxPayload(t *testing.T) {
 
 func TestClosedSlowConsumerWriteDeadline(t *testing.T) {
 	opts := DefaultOptions()
-	opts.WriteDeadline = 10 * time.Microsecond // Make very small to trip.
+	opts.WriteDeadline = 10 * time.Millisecond // Make very small to trip.
```

Comment:
Probably not enough. Bumping it to 30ms in TestWriteDeadline for very same reason.

Reply:
(it was originally at 1 ms though), so that may be ok...

Reply:
Seems to pass on travis..

Reply:
But we can change it..